sdk/publicarchive/downloader.go

```diff
@@ -0,0 +1,215 @@
+// Copyright (C) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package publicarchive
+
+import (
+	"archive/tar"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/ava-labs/avalanche-cli/sdk/network"
+	"github.com/ava-labs/avalanchego/utils/constants"
+	"github.com/ava-labs/avalanchego/utils/logging"
+	"github.com/cavaliergopher/grab/v3"
+	"go.uber.org/zap"
+
+	sdkConstants "github.com/ava-labs/avalanche-cli/sdk/constants"
```

Comment:
inside sdk I shall call this one constants, and the other one cliConstants

Reply:
agree